garden_integration_tests_suite_test.go

```diff
@@ -92,8 +92,12 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	rootfs, exists := os.LookupEnv("TEST_ROOTFS")
 	ExpectWithOffset(1, exists).To(BeTrue(), "Set TEST_ROOTFS Env variable")
 
-	binary, err := gexec.Build("code.cloudfoundry.org/garden-integration-tests/plugins/consume-mem")
-	Expect(err).ToNot(HaveOccurred())
+	binary := ""
+	if runtime.GOOS == "windows" {
+		var err error
+		binary, err = gexec.Build("code.cloudfoundry.org/garden-integration-tests/plugins/consume-mem")
```

Comment:
does windows have go?

Reply:
Windows test runs locally on the VM, whereas in linux, we run this suite as an errand